core/envelope-sim/src/main/java/fr/sncf/osrd/envelope/DriverBehaviour.java

```diff
@@ -0,0 +1,30 @@
+package fr.sncf.osrd.envelope;
+
+public class DriverBehaviour {
+    public double speedTransitionOffsetIncreasing = 0;
+    public double speedTransitionOffsetDecreasing = 0;
```

Comment:
The naming isn't very clear, it's hard to tell what this means without looking at how it's used (though I admit this is difficult).

Why not something like `LowerSpeedLimitAnticipationDistance`, and `higherSpeedLimitDelayDistance`?

And now that I think about it, we're talking about reaction and anticipation. Does it really make sense to use distances rather than times? A "delay distance" is a weird concept. @axrolld

Reply:
I changed the values to `acceleratingPostponementOffset` and `brakingAnticipationOffset`